src/lib/flags.ts

```diff
@@ -76,6 +84,7 @@ const featureFlags: Record<FeatureName, FeatureFlag> = {
   TEST_DIGGING: betaTimeBasedFeatureFlag(new Date("2024-08-01T00:00:00Z")),
   NEW_FRUITS: betaTimeBasedFeatureFlag(new Date("2024-08-01T00:00:00Z")),
   DESERT_PLAZA: betaTimeBasedFeatureFlag(new Date("2024-08-01T00:00:00Z")),
+  PIRATE_CHEST: betaTimeBasedFeatureFlag(new Date("2024-08-16T00:00:00Z")),
```

Comment:
I would suggest using a normal flag here.

`PIRATE_CHEST: defaultFeatureFlag`

The `betaTimeBasedFeatureFlag` are useful when we want something to turn on automatically at a certain time. Usually for seasonal or special event content.